CarpetX/src/io_tsv.cxx

```diff
@@ -209,7 +244,27 @@ void WriteTSVArrays(const cGH *restrict cctkGH, const std::string &filename,
       file << sep << (dir == out_dir ? i : 0);
     const int tl = 0;
     for (int vi = 0; vi < arraygroupdata.numvars; ++vi)
-      file << sep << arraygroupdata.data.at(tl).at(np * vi + DI[out_dir] * i);
+      switch (cgroup.vartype) {
+      case CCTK_VARIABLE_REAL:
+        file << sep
+             << *(CCTK_REAL *)arraygroupdata.data.at(tl).data_at(
```

Comment:
`(const CCTK_REAL *)`

Reply:
Done in [4704af8](https://github.com/EinsteinToolkit/CarpetX/commit/4704af84e360169589e61c330816d0dd9e10f79a)